test-upgrade/2to3/2to3.js

```diff
@@ -0,0 +1,93 @@
+/* globals artifacts */
+/* eslint-disable prefer-arrow-callback */
+import chai from "chai";
+import { checkErrorRevert } from "../../helpers/test-helper";
+
+const namehash = require("eth-ens-namehash");
+
+const IColonyNetwork = artifacts.require("IColonyNetwork");
+const EtherRouter = artifacts.require("EtherRouter");
+const IColony = artifacts.require("IColony");
+const IMetaColony = artifacts.require("IMetaColony");
+const { expect } = chai;
+
+contract("Colony contract upgrade", () => {
+  let metaColony;
+  let colony;
+  let colonyNetwork;
+  let newColonyNetworkResolverAddress;
+
+  const COLONYNETWORK_ADDRESS = "0x5346D0f80e2816FaD329F2c140c870ffc3c3E2Ef";
+  const COLONYNETWORK_OWNER = "0x56a9212f7f495fadce1f27967bef5158199b36c7";
+  const TESTCOLONY_ADDRESS = "0x84bc20B584fA28a278B7a8d5D1Ec5c71224c9f7C";
+  const TESTCOLONY_OWNER = "0xf780ee9f9e50248e7f8433a132a4b0b00b3da313";
+
+  before(async function() {
+    colonyNetwork = await IColonyNetwork.at(COLONYNETWORK_ADDRESS);
+
+    // Upgrade the network to accommodate the new function. The migrations already deployed new versions of everything, so
+    // we can hopefully just piggyback off of that.
+
+    const etherRouter = await EtherRouter.deployed();
+    newColonyNetworkResolverAddress = await etherRouter.resolver();
+    const newlyDeployedNetwork = await IColonyNetwork.at(etherRouter.address);
+    const newColonyVersionResolverAddress = await newlyDeployedNetwork.getColonyVersionResolver(3);
+
+    const metaColonyAddress = await colonyNetwork.getMetaColony();
+    metaColony = await IMetaColony.at(metaColonyAddress);
+    await metaColony.addNetworkColonyVersion(3, newColonyVersionResolverAddress, { from: COLONYNETWORK_OWNER });
+
+    // These tests run against the developers colony. We assume we have forked main chain at 8046998
+    colony = await IColony.at(TESTCOLONY_ADDRESS);
+  });
+
+  describe("check behaviour before colonyNetwork upgrade", function() {
+    it("should not be able to update user orbit db", async function() {
+      await colonyNetwork.registerUserLabel("username", "before update");
```

Comment:
For clarity, maybe bring this line out in the overall `before`? It is relied on in the [second test further down](https://github.com/JoinColony/colonyNetwork/pull/667/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.sol#diff-6cb8c134a81b9080e08341493e6a02c3R61) and it feels cleaner to have it at the top.